jetty-ee9/jetty-ee9-maven-plugin/src/main/java/org/eclipse/jetty/ee9/maven/plugin/QuickStartGenerator.java

```diff
@@ -175,7 +171,7 @@ public void generate()
 
             //save config of the webapp BEFORE we stop
             if (webAppPropsFile != null)
-                WebAppPropertyConverter.toProperties(webApp, webAppPropsFile, contextXml);
+                WebAppPropertyConverter.toProperties(webApp, webAppPropsFile.toFile(), contextXml);
```

Comment:
Avoid going back to `File`.

Reply:
In a different PR focused on the maven plugin